category_import.py

```diff
@@ -0,0 +1,139 @@
+                                )
+                                assets_sub_category = AssetSubCategory.\
+                                    objects.create(
+                                        sub_category_name=row['Sub Category'],
+                                        asset_category=category
+                                    )
+                                assets_sub_category.save()
+                                inserted_records.append([
+                                    assets_sub_category,
+                                    counter]
+                                )
+                                skipped[row['Category']] = [(
+                                    'Category {0} already exists'.
+                                    format(row['Category'])), counter]
+
+                        elif not assets_category:
+                            if assets_sub_category:
+                                asset_category = AssetCategory.objects.create(
+                                    category_name=row['Category']
+                                )
+                                asset_category.save()
+                                inserted_records.append([
+                                    asset_category,
+                                    counter]
+                                )
+                                skipped[row['Sub Category']] = [(
+                                    'Sub Category {0} already exists'.
+                                    format(row['Sub Category'])), counter]
+                            else:
+                                asset_category = AssetCategory.objects.create(
+                                    category_name=row['Category']
+                                )
+                                asset_category.save()
+                                category = AssetCategory.objects.get(
+                                    category_name=row['Category']
+                                )
+                                sub_category = AssetSubCategory.objects.create(
+                                    sub_category_name=row['Sub Category'],
+                                    asset_category=category
+                                )
+                                sub_category.save()
+                                inserted_records.append([
+                                    asset_category,
+                                    counter]
+                                )
+                                inserted_records.append([
+                                    sub_category,
+                                    counter]
+                                )
+
+                        counter += 1
+                        pbar.update(1)
+            print("\n")
+            display_inserted(inserted_records)
+            display_skipped(skipped)
+
+    return 'File not found'
+
+
+def display_inserted(result):
```

Comment:
@QUDUSKUNLE We now have this in a separate file. Pls import from https://github.com/AndelaOSP/art-backend/blob/develop/import_util.py

Reply:
Fixed.